src/test/test_addr.c

```diff
@@ -17,6 +17,8 @@
 #include <sys/un.h>
 #endif
 
+extern int lambada;
```

Comment:
This seems extraneous?

Reply:
Indeed. Wow. I have no memory where that came from.